googleMapsApiModule.py

```diff
@@ -4,27 +4,24 @@
 import mysql.connector
 from constants import BASE_URL
 
-gmaps = googlemaps.Client(config.key)
-#global variable gmaps
+gmaps = googlemaps.Client(config.key)      #global variable gmaps
 
 def direction(origin,destination):
     result = gmaps.directions(origin,destination)
     address = "origin="+origin+"&"+"destination="+destination
```

Comment:
Can we use fstrings here too?

Reply:
@vishakha-lall do you mean at the address section?

Reply:
Yes

Reply:
Changes made. Please review.